lti_tool/models.py

```diff
@@ -466,6 +466,35 @@ def __str__(self):
         return self.title if self.title else self.id_on_platform
 
 
+class LtiLineItem(models.Model):
+    """Represents an LTI LineItem for assignment scoring.
+
+    The line item scope and methods are described in the LTI Assignment and Grade
+    Services 2.0 spec at
+    https://www.imsglobal.org/spec/lti-ags/v2p0#line-item-service-scope-and-allowed-http-methods
+    """
+
+    context = models.ForeignKey(
+        LtiContext,
+        related_name="line_items",
+        on_delete=models.CASCADE,
+        verbose_name=_("context"),
+    )
+    url = models.URLField(_("URL on Platform"))
+    maximum_score = models.FloatField(_("Maximum score"))
+    label = models.CharField(_("Label"), max_length=255)
+    tag = models.CharField(_("Tag"), max_length=255)
+    resource_id = models.CharField(_("Resource ID"), max_length=255)
+    resource_link = models.ForeignKey(
+        LtiResourceLink, on_delete=models.SET_NULL, null=True
+    )
+    start_datetime = models.DateTimeField(_("Start Date/Time"), null=True)
+    end_datetime = models.DateTimeField(_("End Date/Time"), null=True)
```

Comment:
nit: Could we have all lowercase verbose names, except in cased like URL or ID?

Reply:
Happy to oblige.

- [ ] standardize casing of field labels.

Reply:
@michaelwheeler In our discussion yesterday, we had talked about `resource_id` vs. `resource_identifier`. I'm OK with either, though I do see an argument for the more verbose name. Regardless, how do you feel about the use of "ID" in the label vs. "identifier"?

Reply:
It looks like the prevailing style in this module was to use `_id`, as `LtiRegistration.client_id` and `LtiDeployment.deployment_id` exist. I'll keep it.